pkg/cmd/ssh_aws.go

```diff
@@ -284,6 +282,23 @@ func (a *AwsInstanceAttribute) createBastionHostInstance() {
 			capturedOutput, err := captured()
 			checkError(err)
 			a.BastionIP = capturedOutput
+			// check instance-status.status,Values=ok before SSh"
```

Comment:
I tested it with the `instance-status.status check` and it takes a long time until the instance status check is successful. Not sure if we should really add this check because we can ssh onto the instance even before the check is ready. The operators will probably get annoyed by waiting 4-5 minutes to ssh to the instance. WDYT?

Reply:
em, I will double-check it again. 
because last time I failed to use ssh onto the instance even before the check is ready

Reply:
Please double check, I tested it and for me it worked before the status check was succesful, but it is good if you can confirm it as well.

Reply:
yes you are right, It may be network issue when I testing before in APJ region.

Reply:
close it, and Please have review the PR only for bastioninstan name check  https://github.com/gardener/gardenctl/pull/207